lib/assets/components/explore_tiles.dart

```diff
@@ -0,0 +1,182 @@
+import 'package:app/models/Action.dart';
+import 'package:app/models/Campaign.dart';
+import 'package:app/models/Cause.dart';
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+
+class ExploreCampaignTile extends StatelessWidget {
+  final String headerImage;
+  final String title;
+  final ListCause? cause;
```

Comment:
Sure, I'll modify `ListCampaign` and `ListAction`

Reply:
Yeah, my thinking was the way we ensure that is an API requirement rather than a check on the app, do you think that's reasonable? Alternatively, we can gracefully handle the case where it is null but it feel like a bit of a faff given it should always be there

Reply:
Yea I think making it an API requirement is reasonable enough, I'll change the models to assume that `_causes` is always non-empty.

While we're at it, are we keeping the same logic for getting `cause`? (i.e. the first element of `_cause`)

Reply:
Either that or we can just take the first item from the api response in the from json?

Reply:
Can we also now make this non-optional